content/education/manage-coursework-with-github-classroom/get-started-with-github-classroom/index.md

```diff
@@ -9,5 +9,6 @@ redirect_from:
   - /education/manage-coursework-with-github-classroom/get-started-with-github-classroom/basics-of-setting-up-github-classroom
 children:
   - /about-github-classroom
+  - /about-assignments-in-github-classroom
```

Comment:
```suggestion
```

Sarita actually suggested this article goes in the "Teach with GitHub Classroom" section of the docs, and I think that makes sense! That'll require moving the file itself, updating the index file in `education/manage-coursework-with-github-classroom/teach-with-github-classroom`, and updating the filepath on the guides page